frontend/src/components/app/SearchInputField.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+  <v-autocomplete
+    id="suchfeld"
+    v-model="selectedSuggestion"
+    v-model:search="searchQuery"
+    :items="suggestions"
+    density="compact"
+    clearable
+    flat
+    item-title="text"
+    hide-details
+    label="Suche"
+    no-filter
+    prepend-inner-icon="mdi-magnify"
+    return-object
+    variant="solo"
+    @keyup.enter="searchOrShowSelectedSuggestion"
+    @keyup.delete="deleteChar"
+    @update:search="suggest"
+    @click:clear="clearSearch"
+  >
+    <template #no-data>
+      <v-list class="pa-3">
+        <v-list-item-title> Keine Suchvorschläge... </v-list-item-title>
+      </v-list>
+    </template>
+    <template #item="{ props, item, index }">
+      <v-list-item
+        v-bind="props"
+        density="compact"
+        :prepend-icon="iconOfSuggestion(item.raw.type)"
+        :title="item.raw.text"
+        @click="searchOrShowSelectedSuggestion"
+        @focus="updateSearchQuery(index)"
+      />
+    </template>
+    <template #append>
+      <v-tooltip
+        v-model="showtooltip"
+        location="bottom start"
+        :open-on-hover="false"
+      >
+        <template v-slot:activator="{ props }">
+          <v-btn
+            v-bind="props"
+            icon="mdi-information"
+            @click="showtooltip = !showtooltip"
+          >
+          </v-btn>
+        </template>
+        <span>
+          <b>Beispiele, wonach gesucht werden kann:</b><br />
+          * Zählstellnummer / -art (123456, 123456T, ...)<br />
+          * Schlagwörter (Bahn, Isar, Tunnel, Brücke, ...)<br />
+          * Monat (Januar, Februar, ...)<br />
+          * Jahreszeit (Frühling, Sommer, ...)<br />
+          * Projektname / -nummer (U1022, VZ Stadtgrenzen 2019, ...)<br />
+          * Straßen- / Platzname (Rosenheimerplatz, Dachauer Straße, ...)<br />
+          * Datumsbereich (von TT.MM.YYYY bis TT.MM.YYYY)<br />
+          * Messstellennummer / -name (4203,...)<br />
+          <!--            Es fehlt noch: Wetter, Schulzeit, (erweiterte) Zählsituation-->
```

Comment:
Evtl den Kommentar entfernen, da diese Kommentar bei einer Änderung bzw. Implementierung vergessen werden kann und somit vorhanden ist, obwohl in Zukunft dieses Feature bereits implementiert sein könnte.

Reply:
entfernt